modules/ding_place2book/ding_place2book.module

```diff
@@ -416,55 +500,50 @@ function ding_place2book_get_event_state($event_maker_id, $event_id) {
     $p2b = ding_place2book_instance();
     $event = $p2b->getEvent($event_maker_id, $event_id);
     $meta_data = $event->meta_data;
-    $event_state = $meta_data->event_state;
-    $start = strtotime($event->begin_at);
-    $end = strtotime($event->end_at);
+    $event_state = $event->state;
```

Comment:
Some namings has to be changed here. According to [the documentation](http://developer.place2book.com/v1/events/) there is now 3 "states":

- `state`
- `metadata`.`event_state`
- `metadata`.`sales_status`

For future readability and ease of debug, I suggest we follow the namings in the API.

Reply:
We should only use the new state parameter.

Reply:
I know, then let's call it $state instead of $event_state - it confuses the reader.

Reply:
Fixed. Also updated $status to $ticket_link_status to avoid confusion.